lib/Serialization/SerializeSIL.cpp

```diff
@@ -938,7 +938,24 @@ void SILSerializer::writeSILInstruction(const SILInstruction &SI) {
   }
   // SWIFT_ENABLE_TENSORFLOW
   case SILInstructionKind::GraphOperationInst: {
-    llvm_unreachable("Unimplemented");
+    const GraphOperationInst *GI = cast<GraphOperationInst>(&SI);
```

Comment:
Are there any tests for serialization and deserialization for other instructions? If so, may be add some for the graph op as well?

Reply:
I found some tests that do a round trip through serialization. I imitated one of those for graphop!